next.config.js

```diff
@@ -3,6 +3,7 @@ const nextConfig = {
   experimental: {
     appDir: true,
```

Comment:
this can be removed

Reply:
https://github.com/upleveled/next-js-example-winter-2023-vienna-austria/blob/b185b14ff948fe87a99482d0aa749476c3c23826/next.config.js#L3